templates/master/01-master-kubelet/_base/units/kubelet.yaml

```diff
@@ -22,6 +22,10 @@ contents: |
         --container-runtime=remote \
         --container-runtime-endpoint=/var/run/crio/crio.sock \
         --node-labels=node-role.kubernetes.io/master,node.openshift.io/os_id=${ID} \
+{{- if .KubeletIPv6}}
+        --node-ip :: \
```

Comment:
https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/
```
--node-ip string
IP address of the node. If set, kubelet will use this IP address for the node
```

What does it mean to set the node-ip to `::` ?? will this break the apiserver to node communication?

Reply:
@danwinship

Reply:
It requires a not-yet-merged upstream patch to kubelet. https://github.com/kubernetes/kubernetes/pull/85850/files